docs/redirects.csv

```diff
@@ -77,3 +77,5 @@ https://help.expensify.com/articles/expensify-classic/workspace-and-domain-setti
 https://help.expensify.com/articles/expensify-classic/workspace-and-domain-settings/Reimbursement,https://help.expensify.com/articles/expensify-classic/send-payments/Reimbursing-Reports
 https://help.expensify.com/articles/expensify-classic/workspace-and-domain-settings/Tags,https://help.expensify.com/articles/expensify-classic/workspaces/Tags
 https://help.expensify.com/articles/expensify-classic/copilots-and-delegates/User-Roles,https://help.expensify.com/articles/expensify-classic/workspaces/Change-member-workspace-roles
+https://help.expensify.com/articles/expensify-classic/workspace-and-domain-settings/tax-tracking,https://help.expensify.com/articles/expensify-classic/expenses/expenses/Apply-Tax
+https://help.expensify.com/articles/expensify-classic/copilots-and-delegates/User-Roles.html,https://help.expensify.com/expensify-classic/hubs/copilots-and-delegates/
```

Comment:
Are we sure we want to redirect from https://help.expensify.com/articles/expensify-classic/copilots-and-delegates/User-Roles.html to https://help.expensify.com/expensify-classic/hubs/copilots-and-delegates/

Asking as,
- the source link contains html
- And is a valid URL
- And is specific to user roles, while target is general url about /copilots-and-delegates/

Reply:
Yes! I have confirmed with @caitlinwhite1 we want the redirect to go there for now.

Reply:
Thank you for confirming!